src/components/chat/chat.ts

```diff
@@ -0,0 +1,23 @@
+import Block from '../../utils/Block';
+import template from './template.hbs';
+import './style.less';
+
+export interface ChatProps {
+    chatImage?: string;
+    chatAlt?: string;
+    dialogTitle: string;
+    dialogSubText: string;
+    dialogText: string;
+    infoTime: string;
```

Comment:
здесь лучше будет использовать тип Date
```suggestion
    infoTime: Date;
```
Если нужно будет вывести вермя на экран, то удобно будет написать функцию с такой сигнатурой
```
const format = (time: Date): string => {} 
```
которая будет возвращать время в том виде, в котором она должна отображаться